core/trino-main/src/main/java/io/trino/eventlistener/EventListenerManager.java

```diff
@@ -62,11 +63,13 @@ public class EventListenerManager
     private final TimeStat queryCreatedTime = new TimeStat(MILLISECONDS);
     private final TimeStat queryCompletedTime = new TimeStat(MILLISECONDS);
     private final TimeStat splitCompletedTime = new TimeStat(MILLISECONDS);
+    protected final LifeCycleManager lifeCycleManager;
```

Comment:
why is it protected?

Reply:
I needed that in TestingLifecycleManager that inherits. But this is going to change now anyway.